charts/astronomer/templates/registry/registry-configmap.yaml

```diff
@@ -34,6 +34,10 @@ data:
     {{- else }}
       filesystem:
         rootdirectory: /var/lib/registry
+    {{- end }}
+    {{- if (.Values.registry.redirect).disable }}
```

Comment:
why do we need brackets here? does it work without?

Reply:
I think we can remove the brackets since there is a default value set (will try out the scenario), but the original idea was not to avoid a panic when there is no `redirect` section in values.yaml file itself.

Reply:
removed the brackets